composer.json

```diff
@@ -9,10 +9,11 @@
         }
     ],
     "require": {
-        "php": "^7.4",
-        "gitonomy/gitlib": "~1.0",
-        "symfony/console": "^3.3",
-        "code-lts/doctum": "^5.4"
+        "php": "^8.1",
+        "gitonomy/gitlib": "^1.2",
+        "symfony/console": "^6.2",
+        "code-lts/doctum": "^5.5.3",
+        "psr/log": "^3.0"
```

Comment:
Add the following node so that composer installs the correct versions no matter what version you're running locally - require.php alone won't do this

```
"config": {
    "platform": {
        "php": "8.1"
    }
}

Reply:
Make sure you run `composer update -W` after making this change

Reply:
I'm running PHP 8.1 locally. I'd say it's up to developers to make sure they're using the right version locally... we don't tend to do this elsewhere do we? e.g. it's not in https://github.com/silverstripe/demo.silverstripe.org/blob/master/composer.json

Reply:
Mistakes happen. There's no reason not to add this.